src/applications/vaos/actions/newAppointment.js

```diff
@@ -303,26 +340,19 @@ export function updateReasonForAppointmentData(page, uiSchema, data) {
 
 export function openClinicPage(page, uiSchema, schema) {
   return async (dispatch, getState) => {
-    let facilityDetails;
-
     dispatch({
       type: FORM_CLINIC_PAGE_OPENED,
     });
 
-    try {
-      facilityDetails = await getFacilityInfo(
-        getState().newAppointment.data.vaFacility,
-      );
-    } catch (e) {
-      facilityDetails = null;
-    }
+    await dispatch(
+      fetchFacilityDetails(getState().newAppointment.data.vaFacility),
+    );
```

Comment:
Refactored this out into `fetchFacilityDetails()` above. Works fine but I'm not certain that `await`ing this dispatch is the proper way to do things.

Reply:
It's weird, but I don't think it's wrong, necessarily.